pandera/io.py

```diff
@@ -106,18 +134,29 @@ def _serialize_schema(dataframe_schema):
             for index_stats in statistics["index"]
         ]
 
+    if statistics["checks"] is not None:
+        checks = _serialize_dataframe_stats(statistics["checks"])
+
     return {
         "schema_type": "dataframe",
         "version": __version__,
         "columns": columns,
+        "checks": checks,
         "index": index,
         "coerce": dataframe_schema.coerce,
         "strict": dataframe_schema.strict,
     }
 
 
-def _deserialize_check_stats(check, serialized_check_stats, pandas_dtype):
+def _deserialize_check_stats(check, serialized_check_stats, pandas_dtype=None):
+    # pylint: disable=unused-argument
     def handle_stat_dtype(stat):
+        # fixme: change interface to not require a dtype spec
```

Comment:
Same as `_serialize_check_stats`, seems to be fixed?

Reply:
See above.